.clang-format

```diff
@@ -42,16 +42,21 @@ AlwaysBreakTemplateDeclarations: true
 BinPackArguments: true
 BinPackParameters: true
 BraceWrapping:
+#    AfterCaseLabel: true       # added in V10
     AfterClass: true
     AfterControlStatement: true
     AfterEnum: true
+#    AfterExternBlock: true     # added in V10
     AfterFunction: true
     AfterNamespace: false
     AfterStruct: true
     AfterUnion: true
     BeforeCatch: true
     BeforeElse: true
     IndentBraces: false
+#    SplitEmptyFunction: true   # added in V10
+#    SplitEmptyNamespace: true  # added in V10
+#    SplitEmptyRecord: true     # added in V10
```

Comment:
Do we want to add these? If they're useful we can update the docker image (once clang 10 is out), but otherwise these can maybe be left out of this PR?

Reply:
We should leave those out, definitely.